### ISSUE TITLE
How to improve performance? What parameters to customize?

### ISSUE THREAD

**Issue by KSJhon** [xsxjtu/GPPNN#3]
Dear @xsxjtu!

Sorry for interrupting you.
I tried to customize your code to test on Quickbird and Worldview3 satellites.
I generated 8000 pairs of training images, in which the patch size is 128x128 for PAN image and 32x32 for LRMS image.
After training GPPNN using my training set, I tested with the test set, but it works poorly. It is to say, it outperforms other traditional benchmarks but is even worse than early DL work like PanNet.

What parameters can I customize?

Best regards.

**Comment by KSJhon** [comment]
I have set the training epochs to 1000.
Is it the statement <torch.optim.lr_scheduler.StepLR> in your code too strong?
